models/Student.js

```diff
@@ -39,6 +38,10 @@ const StudentSchema = new Schema({
     type: String,
     required: true
   },
+  program_list: {
```

Comment:
This should be a list of ObjectIDs instead of strings. I can't find a great example online but it should be something like [this](https://www.codegrepper.com/code-examples/javascript/type+mongoose.schema.types.objectid+ref+%27user%27).

```js
program_list: [{
  type: Schema.Types.ObjectId,
  ref: 'Program'
}],
```